commit 4a876ce151483edd706753747205b20436a3fca1
Author: Josh Wilsdon <jwilsdon@joyent.com>
Date:   2019-03-13T19:01:42-07:00 (7 months ago)
    
    TRITON-1310 node 6.17 breaks CNAPI
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 63463e0..4cad77c 100644
--- a/Makefile
+++ b/Makefile
@@ -44,7 +44,7 @@ JSSTYLE_FLAGS	= -o indent=4,doxygen,unparenthesized-return=0
 SMF_MANIFESTS_IN = smf/manifests/cnapi.xml.in
 SMF_DTD		= $(REPO_ROOT)/tools/service_bundle.dtd.1
 
-NODE_PREBUILT_VERSION=v6.17.0
+NODE_PREBUILT_VERSION=v6.15.1
 NODE_PREBUILT_TAG=zone
 ifeq ($(shell uname -s),SunOS)
 	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
diff --git a/package.json b/package.json
index c574db9..5e6f81a 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.21.4",
+  "version": "1.21.5",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
